src/lib.rs

```diff
@@ -46,6 +46,8 @@ extern crate anyhow;
 
 extern crate ring;
 
+extern crate async_std;
```

Comment:
Why is it exported? Normally, this crate should be purely internal, as it is an implementation detail